app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,170 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("TaskService 클래스")
+class TaskServiceTest {
+    private TaskService taskService;
+    private static final String TASK_TITLE = "test";
+    private static final Long NOT_EXIST_ID = 999999999L;
+
+    @BeforeEach
+    void setUp() {
+        taskService = new TaskService();
+    }
+
+    @Nested
+    @DisplayName("getTasks 메소드는")
+    class Describe_getTasks {
+        @Nested
+        @DisplayName("등록된 할 일이 없다면")
+        class Context_no_have_tasks {
+            @Test
+            @DisplayName("비어있는 리스트를 리턴한다")
+            void it_returns_emptyTask() {
+                List<Task> tasks = taskService.getTasks();
+                assertThat(tasks).isEmpty();
+            }
+        }
+
+        @Nested
+        @DisplayName("등록된 할 일이 있다면")
+        class Context_have_tasks {
+            @Test
+            @DisplayName("할 일들을 리턴한다")
+            void it_returns_tasks() {
+                Task task1 = new Task();
+                Task task2 = new Task();
+
+                taskService.createTask(task1);
+                taskService.createTask(task2);
+
+                // when
+                List<Task> tasks = taskService.getTasks();
+
+                //then
+                assertThat(tasks.size()).isEqualTo(2);
+                assertThat(tasks).contains(task1, task2);
+            }
+        }
```

Comment:
오 너무너무 잘 작성하셨습니다. 다만 "테스트를 준비하는 코드"는 `@BeforeEach`로 분리해 주시는 것이 좋습니다.

하나의 테스트 코드는 하나의 과학 실험이라고 생각하세요.
실험실 안에서 모든 것을 하려고 하지 않는 것이 좋습니다.
실험 준비는 실험 준비실에서 하고, 실험은 철저하게 오염과 격리된 공간에서 하는 거죠.

```suggestion
        @Nested
        @DisplayName("등록된 할 일이 있다면")
        class Context_have_tasks {
        
            @BeforeEach
            void prepareTasks() {
                // 이 메소드 안에서 실험을 준비합니다.
                Task task1 = new Task();
                Task task2 = new Task();

                taskService.createTask(task1);
                taskService.createTask(task2);             
            }
        
            @Test
            @DisplayName("할 일들을 리턴한다")
            void it_returns_tasks() {
                // 이 메소드 안에서 실험을 실행하고...
                final List<Task> tasks = taskService.getTasks();

                // 실험이 예상대로 실행되었는지 결과를 검증합니다.
                assertThat(tasks.size()).isEqualTo(2);
                assertThat(tasks).contains(task1, task2);
            }
        }
```